### ISSUE TITLE
Breaking change due to `use-bean-validation` and cannot be deactivated

### ISSUE THREAD

**Issue by andrejpetras** [quarkiverse/quarkus-openapi-generator#819]
1. The new `use-bean-validation` feature in version 2.5.0 is enabled by default. Currently, we need to disable it on a per-project as a breaking change to existing source code. I would like to suggest disabling this feature by default, as none breaking change.

```
        this.embeddedTemplateDir = "templates";

        Boolean beanValidation = (Boolean) this.additionalProperties.getOrDefault("use-bean-validation", true);
        this.setUseBeanValidation(beanValidation);
```

Source code: https://github.com/quarkiverse/quarkus-openapi-generator/blob/2.5.0/client/deployment/src/main/java/io/quarkiverse/openapi/generator/deployment/wrapper/QuarkusJavaClientCodegen.java#L56

2. Import of the class `jakarta.validation.Valid` is not disabled when `use-bean-validation` is disabled globally or per client.

```
{! https://github.com/OpenAPITools/openapi-generator/issues/18974 !}
import jakarta.validation.Valid;

{#for imp in imports}
import {imp.import};
```

Source code: https://github.com/quarkiverse/quarkus-openapi-generator/blob/2.5.0/client/deployment/src/main/resources/templates/libraries/microprofile/model.qute#L4

Example breaking PR to update from `2.4.7` to `2.5.0` https://github.com/onecx/onecx-quarkus/pull/214

**Comment by gwydionmv** [comment]
Hello @hbelmiro! We just faced this issue with the breaking change of the mandatory Jakarta import. I can see that it's already fixed but no release has been generated, right? What's the ETA for it? Thanks!

**Comment by ricardozanini** [comment]
@gwydionmv we have another breaking change that I'm waiting to get fixed before releasing a new one: #824
